README.md

```diff
@@ -62,84 +64,45 @@ List of collected metrics is described in [METRICS.md](https://github.com/intels
 ```
-In another terminal window:
 
-Load snap-plugin-collector-df plugin:
-```
-$ $SNAP_PATH/bin/snapctl plugin load snap-plugin-collector-df
+See all available metrics:
 ```
-Load file plugin for publishing:
+$ snapctl metric list
 ```
-$ $SNAP_PATH/bin/snapctl plugin load $SNAP_PATH/plugin/snap-publisher-file
-```
-See available metrics:
 
+Download an [example task file](https://github.com/intelsdi-x/snap-plugin-collector-df/blob/master/examples/tasks/) and load it:
 ```
-$ $SNAP_PATH/bin/snapctl metric list
-```
-
-Create a task manifest file to use snap-plugin-collector-df plugin (exemplary file in [examples/task/] (https://github.com/intelsdi-x/snap-plugin-collector-df/blob/master/examples/task/)):
+$ curl -sfLO https://raw.githubusercontent.com/intelsdi-x/snap-plugin-collector-df/master/examples/tasks/df-file.json
+$ snapctl task create -t df-file.json
+Using task manifest to create task
+Task created
+ID: 480323af-15b0-4af8-a526-eb2ca6d8ae67
+Name: Task-480323af-15b0-4af8-a526-eb2ca6d8ae67
+State: Running
 ```
-{
-    "version": 1,
-    "schedule": {
-        "type": "simple",
-        "interval": "5s"
-    },
-    "workflow": {
-        "collect": {
-            "metrics": {
-                "/intel/procfs/filesystem/*/space_free": {},
-                "/intel/procfs/filesystem/*/space_reserved": {},
-                "/intel/procfs/filesystem/*/inodes_percent_free": {},
-                "/intel/procfs/filesystem/*/inodes_percent_used": {},
-                "/intel/procfs/filesystem/*/device_name": {},
-                "/intel/procfs/filesystem/*/inodes_used": {}
-            },
-            "config": {
-                "/intel/procfs/filesystem": {
-                    "proc_path": "/proc",
-                    "keep_original_mountpoint": true,
-                    "excluded_fs_names": "/proc/sys/fs/binfmt_misc,/var/lib/docker/aufs",
-                    "excluded_fs_types": "proc,binfmt_misc,fuse.gvfsd-fuse,sysfs,cgroup,fusectl,pstore,debugfs,securityfs,devpts,mqueue,hugetlbfs,nsfs,rpc_pipefs,devtmpfs,none,tmpfs,aufs"
-                }
-            },
-            "process": null,
-            "publish": [
-                {
-                    "plugin_name": "file",
-                    "config": {
-                        "file": "/tmp/published_df.log"
-                    }
-                }
-            ]
-        }
-    }
-}
 
-```
-Create a task:
-```
-$ $SNAP_PATH/bin/snapctl task create -t df-file.json
-```
+See realtime output from `snapctl task watch <task_id>` (CTRL+C to exit)
 
 ### Roadmap
 There isn't a current roadmap for this plugin, but it is in active development. As we launch this plugin, we do not have any outstanding requirements for the next release.
 
 If you have a feature request, please add it as an [issue](https://github.com/intelsdi-x/snap-plugin-collector-users/issues) and/or submit a [pull request](https://github.com/intelsdi-x/snap-plugin-collector-users/pulls).
```

Comment:
Could you fix links in this sentence? These links indicate `snap-plugin-collector-users` instead of `snap-plugin-collector-df`.